src/createBlock.ts

```diff
@@ -127,6 +139,7 @@ async function createBlock(): Promise<Block> {
     const data = await createIpfsBlock();
     const formattedData = formatData(data);
     const multiHash = await saveDataToIpfsAsFile(`block-${data.sequence}`, formatData);
+    writeBlockDataToLocalFile(data, multiHash, formattedData);
```

Comment:
This probably should be async, and also all of this probably should have some error handling logic, given that writing to both IPFS and disk can go wrong.

Reply:
i just dropped the whole function in a try-catch. is that wrong? :P